schemas/actions/function/option/types/value.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://raw.githubusercontent.com/bettyblocks/json-schema/master/schemas/actions/function/option/types/value.json",
+  "title": "Options Type Value",
+  "description": "Properties specific to the Value option type.",
+  "type": "object",
+  "properties": {
+    "type": {
+      "const": "Value"
+    },
+    "allowedKinds": {
+      "description": "The options that you want configured for the value.",
+      "type": "array",
+      "items": {
```

Comment:
Maybe you could add `"uniqueItems": true`
https://json-schema.org/understanding-json-schema/reference/array.html#uniqueness